examples/cluster/plot_hdbscan.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+"""
+====================================
+Demo of HDBSCAN clustering algorithm
+====================================
+
+In this demo we will take a look at :class:`sklearn.cluster.HDBSCAN` from the
+perspective of generalizing the :class:`sklearn.cluster.DBSCAN` algorithm.
+We'll compare both algorithms on specific datasets. Finally we'll evaluate
+HDBSCAN's sensitivity to certain hyperparameters. We first define a couple
+utility functions for convenience.
+"""
+
+import numpy as np
+
+from sklearn.cluster import HDBSCAN, DBSCAN
+from sklearn import metrics
+from sklearn.datasets import make_blobs
+import matplotlib.pyplot as plt
+
+
+def plot(X, labels=None, probabilities=None, kwargs=None, ground_truth=False):
```

Comment:
Parameters is less generic:

```suggestion
def plot(X, labels=None, probabilities=None, parameters=None, ground_truth=False):
```

Also, I think `labels` is always passed in, so we can remove the `labels=None` here.